credentials/templates/base.html

```diff
@@ -13,7 +13,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{% block title %}{% endblock title %}</title>
 
-    <link rel="icon" type="image/x-icon" href="/favicon.ico" />
+    <link rel="icon" type="image/x-icon" href="/static/images/favicon.ico" />
```

Comment:
```suggestion
    <link rel="icon" type="image/x-icon" href="{% static "images/favicon.ico" %}" />
```
can we do somthing like this?

Reply:
Done